frontend/src/summary_charts.pug

```diff
@@ -73,28 +85,32 @@
         .summary-chart__title--contribution.mini-font [{{ user.checkedFileTypeContribution }} lines]
         a(
           v-if="filterGroupSelection !== 'groupByRepos'",
-          v-bind:href="getRepoLink(repo[j])", target="_blank",
-          title="click to view repo"
+          v-bind:href="getRepoLink(repo[j])", target="_blank"
         )
-          font-awesome-icon.icon-button(:icon="['fab', 'github']")
+          .tooltip
+            font-awesome-icon.icon-button(:icon="['fab', 'github']")
+            span.tooltip-text Click to view repo
         a(
           v-if="filterGroupSelection !== 'groupByAuthors'",
-          v-bind:href="getAuthorProfileLink(repo[j].name)", target="_blank",
-          title="click to view author's profile"
+          v-bind:href="getAuthorProfileLink(repo[j].name)", target="_blank"
         )
-          font-awesome-icon.icon-button(icon="user")
+          .tooltip
+            font-awesome-icon.icon-button(icon="user")
+            span.tooltip-text Click to view author's profile
         a(
-          title="click to view author's code",
           onclick="deactivateAllOverlays()",
           v-on:click="openTabAuthorship(user, repo, j, isGroupMerged(getGroupName(repo)))"
         )
-          font-awesome-icon.icon-button(icon="code")
+          .tooltip
+            font-awesome-icon.icon-button(icon="code")
+            span.tooltip-text Click to view author's code
```

Comment:
```suggestion
            span.tooltip-text Click to view author's contribution
```
I think this might be more suitable and will work in both the contexts.